dgw_test.go

```diff
@@ -17,7 +17,7 @@ func testPgSetup(t *testing.T) (*sql.DB, func()) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	setupSQL, err := ioutil.ReadFile("./test.sql")
+	setupSQL, err := os.ReadFile("./sql/test.sql")
```

Comment:
[nits]
No modification is needed.

You are concatenating paths directly using "/", but this approach may not work as expected on Windows. In such cases, it is preferable to use `filepath.Join()`.

Reply:
@nao1215 fixed https://github.com/kanmu/dgw/pull/27/commits/ef454dcb7caae3e87d98e580da7870b04bdfe69e